scm-webapp/src/main/java/sonia/scm/lifecycle/SetupContextListener.java

```diff
@@ -115,5 +127,15 @@ private void createAdminAccount() {
       PermissionDescriptor descriptor = new PermissionDescriptor("*");
       permissionAssigner.setPermissionsForUser("scmadmin", Collections.singleton(descriptor));
     }
+
+    private boolean authenticatedGroupDoesNotExists() {
+      return groupManager == null || groupManager.get(AUTHENTICATED) == null;
```

Comment:
`groupManager` is never `null`

Reply:
I forget to include this in the unit tests and for whatever reason came to the conclusion that it would be the best solution. I was wrong.